server/src/main/java/org/opensearch/action/search/SearchUtils.java

```diff
@@ -40,4 +50,52 @@ public static StepListener<BiFunction<String, String, DiscoveryNode>> getConnect
         }
         return lookupListener;
     }
+
+    /**
+     * Delete list of pits, return success if all reader contexts are deleted ( or not found ).
```

Comment:
`Delete list of pit contexts. Returns success only if each reader context is either deleted or not found.`